### ISSUE TITLE
The validation results can not match that claimed in the paper

### ISSUE THREAD

**Issue by voidrank** [kanezaki/pytorch-rotationnet#1]
I followed the instructions. After about 250 epoch, I got some results like this:

Test: [0/40]    Time 41.463 (41.463)    Loss 9.3734 (9.3734)    Prec@1 0.000 (0.000)    Prec@5 0.000 (0.000)
Test: [10/40]   Time 0.369 (11.507)     Loss 9.4063 (9.3822)    Prec@1 0.000 (0.000)    Prec@5 0.000 (0.000)
Test: [20/40]   Time 28.215 (11.223)    Loss 9.3635 (9.3700)    Prec@1 0.000 (0.000)    Prec@5 0.000 (0.000)
Test: [30/40]   Time 3.482 (10.446)     Loss 9.3362 (9.3568)    Prec@1 0.000 (0.000)    Prec@5 100.000 (3.226)

**Comment by kanezaki** [comment]
Could you run your program on a single GPU? You can do that by “export CUDA_VISIBLE_DEVICES=0”. Or, it could be that you linked images improperly. For instance, the directory “ModelNet40_20/train/airplane” should have 1600 images such as “airplane_0001_001.png, airplane_0001_005.png, airplane_0001_009.png, …, airplane_0001_077.png” (20 images per object). If these don’t solve the problem, please try with a smaller learning rate. BTW, I have just uploaded the scripts (using caffe) for reproducing our best results on ModelNet40 and ModelNet10. Please see https://github.com/kanezaki/rotationnet for more details. 😊

**Comment by DilaraAlb** [comment]
@voidrank I have similar results, did you tried with smaller learning rate, or what did you do?
